test/filters.js

```diff
@@ -205,6 +186,13 @@ describe('filters', function () {
       () => test('{{ 183.357 | minus: 12 }}', '171.357'))
     it('should convert first arg as number', () => test('{{ "4" | minus: 1 }}', '3'))
     it('should convert both args as number', () => test('{{ "4" | minus: "1" }}', '3'))
+    it('should return 365 days', function () {
+      var fromDate = new Date("March 31, 2020");
+      var toDate = new Date("March 31, 2021");
+      var src = '{{' + fromDate + '| minus: ' + toDate + '}}';
+      var dst = 365
```

Comment:
this should return a duration not just a plain number

Reply:
Resolved